oaipmh/models.py

```diff
@@ -163,3 +177,74 @@ def update_matterhorn_record(instance, raw, **kwargs):
     _, created = ensure_matterhorn_record(instance)
     if created:
         LOG.info('Created new matterhorn record for %s', instance.identifier)
+
+
+class Series(models.Model):
+    """
+    Record of a Matterhoen (Opencast) series and the playlist things should be published to.
+
+    """
+    class Meta:
+        verbose_name_plural = "Series"
+        unique_together = [
+            # A particular series is unique within a repository
+            ('repository', 'identifier')
+        ]
+
+    identifier = models.CharField(
+        blank=True, max_length=1024,
+        help_text="Identifier for series. Blank for records with no series set"
+    )
+
+    repository = models.ForeignKey(
+        Repository, on_delete=models.CASCADE,
+        help_text="Repository this series was fetched from"
+    )
+
+    title = models.TextField(blank=True, max_length=1024, help_text="Human-readable title")
+
+    playlist = models.ForeignKey(
```

Comment:
Do we want this to be ForeignKey or OnetoOne? ForeingKey will allow one playlist to be used for more than one Series, are we sure we want this? I'd say yes.

Reply:
:+1: